content/docs/core/changelog.mdx

```diff
@@ -17,6 +17,33 @@ Please refer to the [upgrade guide](/docs/core/upgrading) before upgrading.
 
 :::
 
+## [v0.27.2](https://github.com/pomerium/pomerium/tree/v0.27.2) (2024-11-22)
+
+[Full Changelog](https://github.com/pomerium/pomerium/compare/v0.27.1...v0.27.2)
+
+### Pomerium Zero
+
+- Add a Pomerium Zero import tool, allowing you to bring your existing Pomerium configuration into Pomerium Zero.
```

Comment:
I think this should say "Add a Pomerium Core to Zero import tool, allowing..."

Reply:
Proposed: "Introducing a Pomerium Zero import tool, allowing you to..."

Reply:
Added "Pomerium Core to" here.

I think we don't yet have a docs guide page to link to here, but it might be good to come back and add a link once we do.